CHANGELOG

```diff
@@ -1,3 +1,6 @@
+1.0.1:
+    - Add .npmignore
+    - Remove S3 publishing steps
```

Comment:
I actually feel like these are things that aren't worth putting in the CHANGELOG - especially a change that would bump the version. I feel like CHANGELOGs should be more about end user facing changes.